go/host/storage/interfaces.go

```diff
@@ -0,0 +1,62 @@
+package storage
+
+import (
+	"io"
+	"math/big"
+
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ten-protocol/go-ten/go/common"
+	"github.com/ten-protocol/go-ten/go/host/storage/hostdb"
+)
+
+type Storage interface {
+	BatchResolver
+	BlockResolver
+	DatabaseResolver
```

Comment:
is this necessary? It doesn't seem to be used. 
It's a bit of a smell to access the underlying db directly. 
If we can avoid that, then let's get rid of it

Reply:
ah yes, a relic of the past. removed